packages/performance-profiler-plugin/package.json

```diff
@@ -22,17 +22,18 @@
     "dist"
   ],
   "main": "dist/index.js",
-  "types": "dist/typings/index.d.ts",
+  "types": "dist/index.d.ts",
   "scripts": {
-    "build": "cross-env NODE_ENV=development webpack",
-    "build:prod": "cross-env NODE_ENV=production webpack",
-    "build:watch": "cross-env NODE_ENV=development webpack --watch",
+    "build": "tsc -b .",
+    "build:prod": "npm run build",
+    "build:watch": "tsc -b . --watch",
     "clean": "rimraf dist",
     "test": "jest",
     "test:watch": "jest --watch",
     "coverage": "jest --coverage",
     "lint": "tslint --project .",
-    "lint:fix": "tslint --project . --fix"
+    "lint:fix": "tslint --project . --fix",
+    "prebuild": "npm run clean"
```

Comment:
I like the convention of putting the `pre` files above the actual script (in this case before `build`)